### PR TITLE
[Oracle] New Package

### LINKED ISSUE #653: Migrate Oracle audit log package
# Integration release checklist

This checklist is intended for integrations maintainers to ensure consistency
when creating or updating a Package, Module or Dataset for an Integration.

### Migration Checklist

- [x] Change follows the [contributing guidelines](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md)
- [x] Supported versions of the monitoring target are documented
- [x] Generated output for at least 1 log file exists
- [x] Documentation explains how to configure monitoring target to work with the integration
- [x] Fields follow [ECS](https://github.com/elastic/ecs) and [naming conventions](https://www.elastic.co/guide/en/beats/devguide/master/event-conventions.html)
- [x] At least a manual test with ES / Kibana / Agent has been performed.
- [x] Required Kibana version set to:
- [x] Screenshot of the "Add Integration" page on Fleet added to PR

### Dashboards

- [ ] Dashboards exists
- [ ] Screenshots added to manifest
- [ ] Data stream filters added to visualizations

### Data Streams
- database_audit
  - [ ] [Pipeline tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/pipeline_testing.md) added
  - Inputs
    - log
      - [ ] [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) added

### PR DESCRIPTION
**Pull request by P1llus**
## What does this PR do?

Moves the Oracle Audit module from filebeat to integration packages
## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes #653

### CODE REVIEW

File: packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-common-config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+dynamic_fields:
+  event.ingested: ".*"
```
Comment (efd6) [inline packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-common-config.yml:2]: Drop this.

File: packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-oracle-database-audit.log-expected.json [hunk 2]
```diff
@@ -0,0 +1,1168 @@
+{
+    "expected": [
+        {
+            "@timestamp": "2020-10-07T15:58:08.000Z",
+            "event": {
```
Comment (efd6) [inline packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-oracle-database-audit.log-expected.json:5]: Not retaining the original message? Set `preserve_original_event` in the test config?

File: packages/oracle/data_stream/database_audit/elasticsearch/ingest_pipeline/default.yml [hunk 3]
```diff
@@ -0,0 +1,212 @@
+---
+description: Pipeline for parsing Oracle Audit logs
+processors:
+  - set:
+      field: event.ingested
+      value: "{{_ingest.timestamp}}"
```
Comment (efd6) [inline packages/oracle/data_stream/database_audit/elasticsearch/ingest_pipeline/default.yml:6]: Drop.

File: packages/oracle/_dev/build/docs/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Oracle Integration
+
+This integration is for ingesting Audit Trail logs from Oracle Databases.
+
+The integration expects an *.aud audit file that is generated from Oracle Databases by default. If this has been disabled then please see the https://docs.oracle.com/en/database/oracle/oracle-database/19/dbseg/introduction-to-auditing.html#GUID-8D96829C-9151-4FA4-BED9-831D088F12FF[Oracle Database Audit Trail Documentation].
```
Comment (efd6) [inline packages/oracle/_dev/build/docs/README.md:5]:
```suggestion
The integration expects an *.aud audit file that is generated from Oracle Databases by default. If this has been disabled then please see the [Oracle Database Audit Trail Documentation](https://docs.oracle.com/en/database/oracle/oracle-database/19/dbseg/introduction-to-auditing.html#GUID-8D96829C-9151-4FA4-BED9-831D088F12FF).
```

File: packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-common-config.yml (hunk 1 above)
Comment (P1llus) [reply packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-common-config.yml:2]: This package is for 7.x and 8.x though?

File: packages/oracle/data_stream/database_audit/elasticsearch/ingest_pipeline/default.yml (hunk 3 above)
Comment (P1llus) [reply packages/oracle/data_stream/database_audit/elasticsearch/ingest_pipeline/default.yml:6]: This package is for 7.x and 8.x though?

**Review by efd6 — APPROVED** [review]

**Pull request merged**